### PR TITLE
style(suite): refactor assetCard and assetRow so the Coin and Name are separated and reusable in different layouts

### PR DESCRIPTION
**Pull request by peter-sanderson**
Addressing this comment: https://github.com/trezor/trezor-suite/pull/10636

### CODE REVIEW

File: packages/components/src/components/Card/Card.tsx [hunk 1]
```diff
@@ -19,6 +19,24 @@ const Wrapper = styled.div<{ $elevation: Elevation; $paddingSize: number }>`
...
     }};
+
+    ${({ onClick, theme }) =>
+        onClick !== undefined
+            ? css`
+                  :hover {
+                      cursor: pointer;
+
+                      box-shadow: ${() => {
+                          const boxShadow = mapElevationToBoxShadow[3]; // Todo: Hardcoded for now, till we have correct elevations (shall be `nextElevation[$elevation]`)
```
Comment (peter-sanderson) [inline packages/components/src/components/Card/Card.tsx]: replace this with `shadowElevated` directly

File: packages/components/src/components/Card/Card.tsx [hunk 2]
```diff
@@ -19,6 +19,24 @@ const Wrapper = styled.div<{ $elevation: Elevation; $paddingSize: number }>`
...
+
+                          return boxShadow !== undefined ? theme[boxShadow] : undefined;
+                      }};
+                  }
+
+                  transition: box-shadow 0.2s;
```
Comment (dahaca) [inline packages/components/src/components/Card/Card.tsx]: put it outside of the condition, as like this it's re-written by the one on the line 41.

File: packages/suite/src/views/dashboard/components/AssetsView/components/AssetCoinLogo.tsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        </LogoWrapper>
+    );
+};
+
+export const AssetCoinLogoSkeleton = () => {
+    const { elevation } = useElevation();
+
+    return (
+        <LogoWrapper>
+            <SkeletonCircle size={44} elevation={elevation} />
```
Comment (dahaca) [inline packages/suite/src/views/dashboard/components/AssetsView/components/AssetCoinLogo.tsx]: No need for `elevation` in skeletons anymore :)

File: packages/suite/src/views/dashboard/components/AssetsView/components/AssetCardInfo.tsx [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    animate?: boolean;
+}
+
+const Flex = styled.div`
+    display: flex;
+`;
+
+const Container = styled.div`
+    display: flex;
+`;
```
Comment (dahaca) [inline packages/suite/src/views/dashboard/components/AssetsView/components/AssetCardInfo.tsx]: you can keep only 1 ;)

File: packages/suite/src/views/dashboard/components/AssetsView/components/AssetTableHeader.tsx [hunk 5]
```diff
@@ -20,20 +21,25 @@ const Header = styled.div`
...
-            <Translation id="TR_7D_CHANGE" />
-        </Header>
-        <Header /> {/* Buy Button */}
-    </AssetTableRowGrid>
-);
+export const AssetTableHeader = () => {
+    const { elevation } = useElevation();
+
+    return (
+        <AssetTableRowGrid elevation={elevation}>
```
Comment (dahaca) [inline packages/suite/src/views/dashboard/components/AssetsView/components/AssetTableHeader.tsx]: nit: could be a component with the hook, not the styled component, just like the skeletons.

File: packages/components/src/components/Card/Card.tsx (hunk 2 above)
Comment (peter-sanderson) [reply packages/components/src/components/Card/Card.tsx]: :heavy_check_mark:

File: packages/suite/src/views/dashboard/components/AssetsView/components/AssetCardInfo.tsx (hunk 4 above)
Comment (peter-sanderson) [reply packages/suite/src/views/dashboard/components/AssetsView/components/AssetCardInfo.tsx]: lol  :D

Comment (peter-sanderson) [reply packages/suite/src/views/dashboard/components/AssetsView/components/AssetCardInfo.tsx]: :heavy_check_mark:

File: packages/suite/src/views/dashboard/components/AssetsView/components/AssetTableHeader.tsx (hunk 5 above)
Comment (peter-sanderson) [reply packages/suite/src/views/dashboard/components/AssetsView/components/AssetTableHeader.tsx]: Good point :heavy_check_mark:

File: packages/suite/src/views/dashboard/components/AssetsView/components/AssetCoinLogo.tsx (hunk 3 above)
Comment (peter-sanderson) [reply packages/suite/src/views/dashboard/components/AssetsView/components/AssetCoinLogo.tsx]: yup, its typeerror anyway, resolving

**Review by dahaca — APPROVED** [review]

**Pull request merged**